presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -623,11 +669,13 @@ private static int ceil(int dividend, int divisor)
     {
         private final OrcDataSourceId sourceId;
         private final long offset;
+        private final int ordinal;
 
-        public StripeId(OrcDataSourceId sourceId, long offset)
+        public StripeId(OrcDataSourceId sourceId, long offset, int ordinal)
```

Comment:
I need the stripeOrdinal (= absolute stripe number) to find the offset + length of the footer/index in the metadata cache slice.